.github/workflows/linter.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow executes several linters on changed files based on languages used in your code base whenever
+# you push a code or open a pull request.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/github/super-linter
+name: Lint Code Base
+
+on:
+  push:
+    branches: [ task/lint-after-push-dcp-682 ]
+  pull_request:
+    branches: [ task/test-linting ]
+  workflow_dispatch:
+    branches: [ task/lint-after-push-dcp-682 ]
+
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
```

Comment:
I think woud be better just to lint changes not the whole repo

Reply:
That is what happens.